lib/ASTGen/Sources/ASTGen/Bridge.swift

```diff
@@ -167,10 +167,14 @@ extension TokenSyntax {
   /// - Parameter astgen: The visitor providing the `ASTContext`.
   @inline(__always)
   func bridgedIdentifier(in astgen: ASTGenVisitor) -> BridgedIdentifier {
-    var text = self.text
-    return text.withBridgedString { bridged in
-      astgen.ctx.getIdentifier(bridged)
+    var text = self.rawText
+    if rawText == "_" {
+      return nil
+    }
```

Comment:
Can this now check for `tokenKind == .wildcard` instead?

Reply:
`tokenKind` instantiate `String`.  Let's expose `TokenSyntax.tokenView.rawKind` as `TokenSyntax.rawTokenKind` as a SPI, in followups.